spec_helper.rb

```diff
@@ -12,14 +12,17 @@
   $stdout = File.open("/tmp/fastlane_tests", "w")
 end
 
-xcode_path = FastlaneCore::Helper.xcode_path
-unless xcode_path.include?("Contents/Developer")
-  UI.error("Seems like you didn't set the developer tools path correctly")
-  UI.error("Detected path '#{xcode_path}'") if xcode_path.to_s.length > 0
-  UI.error("Please run the following on your machine")
-  UI.command("sudo xcode-select -s /Applications/Xcode.app")
-  UI.error("Adapt the path if you have Xcode installed/named somewhere else")
-  exit(1)
+is_mac = FastlaneCore::Helper.is_mac?
```

Comment:
It feels like assigning this value, only to use it one is overhead, we can just have that right next to the `if` keyword? :)

Reply:
Sure we can - leftover from my wrangling with `unless`